### ISSUE TITLE
creating separate processes meaning?

### ISSUE THREAD

**Issue by anilbandrapalli** [OptimalBits/bull#2349]
Hi, i am new to this package. in the bull package mentioned that separate processes for running the process function. meaning do we need to keep this function in separate file in the same node js application or with this function alone create new node js application all together?

Please clarify me on this.

**Comment by manast** [comment]
I am not understanding your question, can you rephrase it? what do you mean by "application"?

**Comment by anilbandrapalli** [comment]
Hi manast, sorry i couldn't give you proper explanation. what i mean is , "separate processes for running the process function" means do i need to maintain 2 nodejs applications/servers (one for responsible for accepting the jobs and another server for running process function meaning executing the jobs). Is my understanding correct ?

**Comment by manast** [comment]
You do not need to, but normally you have several workers on different servers to achieve scalability.

**Comment by anilbandrapalli** [comment]
Thank you for the clarification
